bucket/postgresql14.json

```diff
@@ -0,0 +1,45 @@
+{
+    "version": "14.6",
+    "description": "Object-relational database management system based on POSTGRES. (version 14)",
+    "homepage": "https://www.postgresql.org",
+    "license": "PostgreSQL",
+    "notes": [
+        "Run run 'pg_ctl start' or 'pg_ctl stop' to start and stop the database or",
+        "register it as a service by running 'pg_ctl register -N PostgreSQL' from an elevated shell.",
+        "Default superuser login: postgres, password: <blank>"
+    ],
+    "architecture": {
+        "64bit": {
+            "url": "https://get.enterprisedb.com/postgresql/postgresql-14.6-1-windows-x64-binaries.zip",
+            "hash": "fb2cb01b07a952f578c57fbe20112e8f1e3722117c68b3bb2c3290a276005f94"
+        }
+    },
+    "extract_dir": "pgsql",
+    "post_install": [
+        "if (!(Test-Path \"$dir\\data\\pg_hba.conf\")) {",
+        "   Invoke-ExternalCommand -FilePath \"$dir\\bin\\initdb.exe\" -ArgumentList @('--username=postgres', '--encoding=UTF8', '--locale=en', '--lc-collate=C') | Out-Null",
+        "}"
+    ],
+    "shortcuts": [
+        [
+            "pgAdmin 4\\bin\\pgAdmin4.exe",
+            "pgAdmin 4"
+        ]
+    ],
+    "env_add_path": "bin",
+    "env_set": {
+        "PGDATA": "$dir\\data"
+    },
+    "persist": "data",
+    "checkver": {
+        "url": "https://www.postgresql.org/ftp/source/",
```

Comment:
Should we be checking
https://www.enterprisedb.com/download-postgresql-binaries
instead, due to the `-1` suffix?

Reply:
🤷 this is essentially a copy of the `postgresql13` manifest